collect/cnrfc/cnrfc.py

```diff
@@ -539,7 +539,8 @@ def download_watershed_file(watershed, date_string, forecast_type, duration=None
 
     # set path for case where path set to None
     if path is None:
-        path = url.split('/')[-1].replace('.zip', '.csv')
+        filename = url.split('/')[-1].replace('.zip', '.csv')
+        path = os.path.join(os.getcwd(), filename)
```

Comment:
`os.getcwd()` is dependent on the location of where your script is executed, rather than where your script is located. This is totally valid, but we were trying to implement the behavior where the files would be created in a location relative to the script location.  Could an alternative be to check if `os.path.dirname` is giving us a valid directory before trying to create it?

Reply:
Something like this [c78ff06](https://github.com/MBKEngineers/collect/pull/97/commits/c78ff064b8b163d77f6685156439050f7f3203e6)?

Reply:
Exactly.  Could possibly set a folder variable to represent `os.path.dirname(path)` to avoid the number of times you have to call it.  Honestly, that won't reduce the number of lines though so I'll leave that up to you.

Reply:
I added the variable in [aef01aa](https://github.com/MBKEngineers/collect/pull/97/commits/aef01aa8cc0bb5be5987440414c5240812494b04)! I think we're both fans of easier-to-understand-even-if-longer code.

Reply:
Great! Thanks for that adjustment :)